### PR TITLE
Move proc_info.js to use proc_info.template

### PR DESCRIPTION
**Pull request by esbie**
Instead of using jquery to add render process metrics, we now can
use the proc_info.template that rmars added.

### CODE REVIEW

File: linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/summary.js [hunk 1]
```diff
@@ -18,18 +18,20 @@ Handlebars.registerHelper('pluralize', function(number, single, plural) {
...
   $.get("/admin/metrics.json")
-).done(function(interfacesRsp, requestStatsRsp, metricsJson) {
+).done(function(processInfoRsp, interfacesRsp, requestStatsRsp, metricsJson) {
   var ifacesTemplate = Handlebars.compile(interfacesRsp[0]),
       summaryTemplate = Handlebars.compile(requestStatsRsp[0]),
       routers = Routers(metricsJson[0]),
       namers = Namers(metricsJson[0]);
 
   $(function() {
+    console.log($("#linkerd-version"))
```
Comment (rmars) [inline linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/summary.js]: errant console.log :)

**Comment by rmars** [comment]
:star: :clipboard:

File: linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/process_info.js [hunk 2]
```diff
@@ -17,26 +25,30 @@ var ProcInfo = (function() {
...
-    _(data).each(function(obj) {
-      var id = obj.name.replace(/\//g, "-");
-      var value = pretty(obj.name, obj.value);
-      $("#"+id).text(value);
+  function render($root, data) {
+    var templateData = _.map(stats, function(stat) {
+      if (stat.dataKey) {
+        var obj = _.find(data, ["name", stat.dataKey]);
+        var value = pretty(obj.name, obj.value);
+        return _.merge({}, stat, {value: value});
```
Comment (klingerf) [inline linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/process_info.js]: Do you need there '{}' argument here? I'd expect merge to work with `_.merge(stat, {value: value})`.

Comment (klingerf) [inline linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/process_info.js]: Oh, maybe you don't want to modify `stat`?

**Comment by klingerf** [comment]
:star: :sparkler: Looks good to me, but I think it (now) needs a rebase.

File: linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/process_info.js (hunk 2 above)
Comment (esbie) [inline linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/process_info.js]: that was my original thinking, although looking at this a second time, it's probably fine to alter stat (and maybe even preferable for less object garbage)

**Pull request merged**